luxonis_train/attached_modules/metrics/confusion_matrix/fomo_confusion_matrix.py

```diff
@@ -0,0 +1,111 @@
+from typing import Optional
+
+import torch
+from loguru import logger
+from torch import Tensor
+from typing_extensions import override
+
+from luxonis_train.tasks import Tasks
+
+from .detection_confusion_matrix import DetectionConfusionMatrix
+from .utils import compute_mcc
+
+
+class FomoConfusionMatrix(DetectionConfusionMatrix):
+    supported_tasks = [Tasks.FOMO]
+
+    def __init__(self, iou_threshold: Optional[float] = None, **kwargs):
+        if iou_threshold is not None and iou_threshold != 0.0:
+            logger.warning(
+                "The `iou_threshold` parameter is ignored for FomoConfusionMatrix and is hardcoded to 0. "
+                "This is by design to align with FOMO's use case.",
+            )
+        super().__init__(iou_threshold=0.0, **kwargs)
+
+    @override
+    def update(
+        self,
+        keypoints: list[Tensor],
+        target_boundingbox: Tensor,
+    ) -> None:
+        """Override update to convert FOMO keypoints into bounding boxes
+        before calling the parent update method."""
+        pred_bboxes = self._keypoints_to_bboxes(
+            keypoints,
+            self.original_in_shape[1],
+            self.original_in_shape[2],
+        )
+
+        super().update(pred_bboxes, target_boundingbox)
+
+    @override
+    def compute(self) -> Tensor:
```

Comment:
The `compute` method doesn't need to be overridden (it does the same thing as `DetectionConfusionMatrix.compute`